.github/workflows/build.yaml

```diff
@@ -135,6 +135,7 @@ jobs:
         include:
           - cfg: {name: 'ubuntu20',      base: 'ubuntu20', builddir: 'ubuntu20'}
           - cfg: {name: 'ubuntu22',      base: 'ubuntu22', builddir: 'ubuntu22'}
+          - cfg: {name: 'asserton',      base: 'asserton', builddir: 'asserton'}
```

Comment:
why `asserton`, not `assertion`? Name length restriction?

Reply:
FWIW, I read it as `assert on`

Reply:
Haha, I read it as `asser` `ton`.